src/Api/Controllers/FoldersController.cs

```diff
@@ -0,0 +1,21 @@
+using Application.Common.Models;
+using Application.Common.Models.Dtos.Physical;
+using Application.Folders.Commands.AddFolder;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+public class FoldersController : ApiControllerBase
+{
+    [HttpPost]
+    [ProducesResponseType(StatusCodes.Status200OK)]    
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status409Conflict)]
+    public async Task<ActionResult<Result<FolderDto>>> AddFolders([FromBody] AddFolderCommand command)
```

Comment:
Plural form?

Reply:
fixed at https://github.com/ProFile-org/ProFile-BE/pull/80/commits/6f136a5a8d36c9c5f790d5718e77bde1acd59818